micromamba/tests/test_remove.py

```diff
@@ -64,7 +61,7 @@ def test_remove_orphaned(tmp_home, tmp_root_prefix, tmp_xtensor_env, tmp_env_nam
     # assert len(res["actions"]["UNLINK"]) == len(env_pkgs) + (
     assert len(res["actions"]["UNLINK"]) == 3 + (
```

Comment:
```suggestion
    assert len(res["actions"]["UNLINK"]) ==  len(env_pkgs) + (
```

Reply:
Unfortunately, those changes do not work. :confused: 

```diff
diff --git i/micromamba/tests/test_remove.py w/micromamba/tests/test_remove.py
index f149c4b3..b7bc6e75 100644
--- i/micromamba/tests/test_remove.py
+++ w/micromamba/tests/test_remove.py
@@ -58,10 +58,9 @@ def test_remove_orphaned(tmp_home, tmp_root_prefix, tmp_xtensor_env, tmp_env_nam
     assert res["success"]
     # TODO: find a better use case so we can revert to len(env_pkgs) instead
     # of magic number
-    # assert len(res["actions"]["UNLINK"]) == len(env_pkgs) + (
-    assert len(res["actions"]["UNLINK"]) == 3 + (
+    assert len(res["actions"]["UNLINK"]) == len(env_pkgs) + (
         1 if helpers.dry_run_tests == helpers.DryRun.DRY else 0
-    ) + (platform.system() == "Linux")  # xtl is not removed on Linux
+    )
     for p in res["actions"]["UNLINK"]:
         assert p["name"] in env_pkgs
     assert res["actions"]["PREFIX"] == str(tmp_xtensor_env)
```

```
E       AssertionError: assert 4 == (9 + 0)
E        +  where 4 = len([{'build': 'h00ab1b0_0', 'build_number': 0, 'build_string': 'h00ab1b0_0', 'channel': 'https://conda.anaconda.org/conda-forge/linux-64', ...}, {'build': 'h00ab1b0_0', 'build_number': 0, 'build_string': 'h00ab1b0_0', 'channel': 'https://conda.anaconda.org/conda-forge/linux-64', ...}, {'build': 'h4852527_1', 'build_number': 1, 'build_string': 'h4852527_1', 'channel': 'https://conda.anaconda.org/conda-forge/linux-64', ...}, {'build': 'hc0a3c3a_1', 'build_number': 1, 'build_string': 'hc0a3c3a_1', 'channel': 'https://conda.anaconda.org/conda-forge/linux-64', ...}])
E        +  and   9 = len(['_libgcc_mutex', '_openmp_mutex', 'libgcc', 'libgcc-ng', 'libgomp', 'libstdcxx', ...])
```